# Will the node publish localization messages to reset amcl if possible?
bool publishing_localization

# How bad the current localization is
float32 localization_badness

# How many successful navigations have been observed so far
uint32 successful_navs